include/swift/AST/LayoutConstraint.h

```diff
@@ -170,6 +196,18 @@ class LayoutConstraintInfo : public llvm::FoldingSetNode {
 
   static bool isNativeRefCountedObject(LayoutConstraintKind Kind);
 
+  static bool isAnyRefCountedObject(LayoutConstraintKind Kind);
```

Comment:
I don't see the reason for these `static` versions. They just seem to bloat the API.